releasenotes/notes/bosonic-operator-and-mapper-45bfde873f092681.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Introduced a new feature that implements the bosonic operators ``BosonicOp``.
+    Its functionalities are analogous to the ``FermionicOp`` It should be used to represent a bosonic
+    operator, so if one wants to represent the boson number operator it should do for example::
+
+      from qiskit_nature.second_q.operators import BosonicOp
+      bosonic_op = BosonicOp({'+_0 -_0': 1}, num_modes=1)
+
+    The main differences with respect to the fermionic operator are that this class uses the commutator
+    relations instead of the anti-commutator (due to the nature of the particles) and that the simplify
+    method ``bosonic_op.simplify()`` does not commute with the index order ``bosonic_op.index_order()``
+
+  - |
+    In order to use the bosonic operator in a quantum circuit, this featrue also introduces the
+    bosonic linear mapper, which allows to map the BosonicOp to the qubit space. This mapper is based
+    on the paper: A. Miessen et al., Quantum algorithms for quantum dynamics: A performance study on the
+    spin-boson model, Phys. Rev. Research 3, 043212.
```

I think that the reno can just directly have a link to the paper :)

```suggestion
    on `this paper <https://link.aps.org/doi/10.1103/PhysRevResearch.3.043212>`_
```

---

Solved